daemon/graphdriver/driver.go

```diff
@@ -277,6 +277,15 @@ func scanPriorDrivers(root string) map[string]bool {
 	return driversMap
 }
 
+// IsInitialized checks if the driver's home-directory exists and is non-empty.
+func IsInitialized(driverHome string) bool {
+	if _, err := os.Stat(driverHome); os.IsNotExist(err) {
+		return false
```

Comment:
It's not quite clear what to do with other errors from `os.Stat()`. On a normal system there should not be any, but if there is an error, it makes sense to at least report it to make debugging easier.

So, something like
```go
if err != nil {
   logrus.Warnf("graphdriver.IsInitialized: stat failed: %v", err)
}
```